assets/prototype/domain/shared/predicates/datetimes/filters/belowCapacity/index.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * External dependencies
+ */
+import { is } from 'ramda';
+
+/**
+ * Internal dependencies
+ */
+import capacityBelow from './capacityBelow';
+import isTrashed from '../../isTrashed';
+import { Datetime } from '../../../../../eventEditor/data/types';
+
+type BelowCapacityProps = {
+	capacity: number;
+	dates: Datetime[];
+};
+
+const belowCapacity = ({ capacity, dates }: BelowCapacityProps): Datetime[] | [] => {
+	const filterFn = (date: Datetime) => {
+		return !isTrashed(date) && capacityBelow({ capacity, date });
```

Comment:
what's the difference between and reasons for having `belowCapacity` AND `capacityBelow` ???

Reply:
Good question. A few things happened here.
1. I didn't liked the mumbo jumbo from the right side and have done the extraction from the left side:
![image](https://user-images.githubusercontent.com/1477453/72056290-02864c00-32d5-11ea-8f8c-729f4c2afca0.png)

Reply:
2. Instead of extracting that into `below50Capacity` function, I've decided to go with a more generic `belowCapacity`.
and
3. I also wanted to leave `belowCapacity` super simple and this is why I extracted further with `capacityBelow`, which I've just renamed to just `filter`, to avoid confusion, open for a better naming.
***
https://github.com/eventespresso/event-espresso-core/pull/2174/commits/576e543f46a725bf5646e92a069a46c1b1e3a0e7